File: cfgs/java_cfgs_foo/COUNT_POSSIBLE_PATHS_TOP_LEFT_BOTTOM_RIGHT_NXM_MATRIX_2.java
Nodes
1;special;;;
2;statement;int[] dp = new int[foo_1];15;
3;statement;foo_1[0] = 1;16;
4;statement;int i = 0;17;
5;conditional;foo_1 < foo_2;18;
6;statement;int j = 1;20;
7;conditional;foo_1 < foo_2;21;
8;statement;foo_1[foo_2] += foo_3[foo_4 - 1];23;
9;statement;j++;22;
10;statement;i++;19;
11;statement;return foo_1[foo_2 - 1];26;
12;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;11;
5;7;
6;8;10;
7;9;
8;7;
9;5;
10;12;
11;

File: cfgs/pythoncfgs/COUNT_POSSIBLE_PATHS_TOP_LEFT_BOTTOM_RIGHT_NXM_MATRIX_2.py
Nodes
1;statement;foo_1 = [1 for foo_2 in fun_1(foo_3)];8;
2;conditional;fun_1(foo_1 - 1);9;
3;statement;return foo_1[foo_2 - 1];12;
4;conditional;foo < foo;10;
5;statement;foo_1[foo_2] += foo_3[foo_4 - 1];11;
6;exit;;
Edges
0;2;
1;3;4;
2;6;
3;2;5;
4;4;
5;
